### DISCUSSION TITLE
copy / paste from folders

### DISCUSSION THREAD

**Discussion by Logic-Bits** [rustdesk/rustdesk#1262] [General]
Thanks for the awesome work so far!

I took # 6 from https://github.com/rustdesk/rustdesk/issues/1168 and put it in a seperate Feature Request.

6) (not part of the File Transfer dialog) Allow copy/paste a file from the client PC directly into a folder on the remote PC
I'm guessing this one is more difficult to implement but it would be a "nice to have" feature. Basically, allow copying a file or folder(s) from Windows Explorer in the client PC, then paste them directly into a folder in Windows Explorer on the remote PC. When pressing Ctrl+V/right click > Paste/etc at the destination, this would automatically start a file transfer of the copied files from the appropriate source/to the appropriate destination, whether it's to/from the local/remote machine.

**Comment by Logic-Bits** [comment]
just saw this works - it needs to be enabled thru the menu tho.

![grafik](https://user-images.githubusercontent.com/2421123/185418520-98d1bbfa-1cc3-42c3-aae9-b62c0f81e873.png)

I guess this can be closed then. Or maybe it should be enabled by default (I think TeamViewer does it)?

**Reply by ertankucukoglu** [reply]
> just saw this works - it needs to be enabled thru the menu tho.
> 
> ![grafik](https://user-images.githubusercontent.com/2421123/185418520-98d1bbfa-1cc3-42c3-aae9-b62c0f81e873.png)

Where did you see this menu? I could not find it on a recent nightly build

**Comment by shekharmax** [comment]
Where you find this menu allow file copy and paste
![image](https://user-images.githubusercontent.com/108380202/203265966-16acc5a5-32d2-4603-8ccc-f69375dc0e77.png)
